scripts/core/helpers/generic.js

```diff
@@ -0,0 +1,20 @@
+// https://gist.github.com/jed/982883
+export function uuid(a) {
```

Comment:
I would call this file rather `helpers/uuid.js`, generic doesn't say much about what is in there

Reply:
I was planning to dedicate the file to stateless functions without context, but it may be a better idea to just name it `uuid` and we'll create some folders for discoverability when there are too much files.